packages/amplify-provider-awscloudformation/lib/amplify-metric-emitter.js

```diff
@@ -0,0 +1,78 @@
+const awsClient = require('../src/aws-utils/aws');
+const constants = require('./constants');
+const moment = require('moment');
+const crypto = require('crypto');
+const fs = require('fs');
+
+const kinesisStreamName = 'aws_amplify_metrics_user_stream';
+const kinesisAmplifyRoleArn = 'arn:aws:iam::827149277658:role/aws-amplify-cli-metrics-putevent-role';
```

Comment:
Is it safe to expose these?

Reply:
Yup..discussed offline. This PR will go through an AppSec review as well, so this could be looked at my the security team.

Reply:
ok